{"uuid":"4560a2c5-dfc9-4120-a038-409895ba587a","name":"Scenario Outline: To validate if user getting error message when enter valid python code in operations stack page","children":["c2b6bc1d-0502-4a7c-835f-0244c3fc1244"],"befores":[],"afters":[{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065854245,"stop":1709065854246},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065854857,"stop":1709065854858},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065857272,"stop":1709065857273},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065857774,"stop":1709065857780},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065858222,"stop":1709065858223},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065858862,"stop":1709065858865},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065859676,"stop":1709065859676},{"name":"stepDefinitions.ApplicationHooks.tearDown()","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065859677,"stop":1709065860288}],"start":1709065850547,"stop":1709065860288}